### PR TITLE
reordered demo test

### PR DESCRIPTION
**Pull request by adrianmaciuc**
move test from folder to folder

### CODE REVIEW

**Review by simonleventeattila — APPROVED** [review]
looks good

**Comment by AdrianCiuciui** [comment]
no comment

**Comment by simonleventeattila** [comment]
Loren ipsum

**Review by RaduMoldovan30 — COMMENTED** [review]
Congrats, Adi

File: tests/demo-todo-app.spec.js [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+test.beforeEach(async ({ page }) => {
+  await page.goto("https://demo.playwright.dev/todomvc");
+});
+
+const TODO_ITEMS = [
+  "buy more cheese",
+  "feed the cat",
+  "book a doctors appointment",
+];
+
```
Comment (RaduMoldovan30) [inline tests/demo-todo-app.spec.js:13]: Good job

**Review by RaduMoldovan30 — COMMENTED** [review]
Done

File: tests/demo-todo-app.spec.js (hunk 1 above)
Comment (RaduMoldovan30) [reply tests/demo-todo-app.spec.js:13]: You have done a great job

**Review by RaduMoldovan30 — COMMENTED** [review]
Done

**Comment by simonleventeattila** [comment]
Clean-up

**Pull request closed without merging**